packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -138,6 +138,18 @@ bool _isWhitespace(int codeUnit) {
   return true;
 }
 
+/// Returns true if [codeUnit] is a leading (high) surrogate for a surrogate
```

Comment:
You can use comment `//` here.

Reply:
What is the disadvantage to using doc comments for private methods? (Dart style soft-recommends this practice; I know we don't necessarily follow Dart style, but unless there's a reason not to it seems odd.)

Reply:
I don't have strong preference. We general uses comment to describe private classes or properties (at least in `flutter/packages/flutter`).